geth/node/manager.go

```diff
@@ -469,31 +486,270 @@ func (m *NodeManager) AccountKeyStore() (*keystore.KeyStore, error) {
 }
 
 // RPCClient exposes reference to RPC client connected to the running node.
-func (m *NodeManager) RPCClient() *rpc.Client {
-	m.Lock()
-	defer m.Unlock()
+func (m *NodeManager) RPCClient() geth.RPCClient {
+	return m.getRPCClient()
+}
 
-	return m.rpcClient
+// GetStatusBackend exposes StatusBackend interface.
+func (m *NodeManager) GetStatusBackend() (services.StatusBackend, error) {
+	_, err := m.getLesService()
```

Comment:
It's extremely confusing that `m.getLesService` needs to be called to provide `m.statusBackend`.